### PR TITLE
Update youtube extension

### LINKED ISSUE #7086: [YouTube] Display video duration
### Extension

https://www.raycast.com/tonka3000/youtube

### Description

Ability to display video duration for the Popular Videos and Search Videos commands.
Display video duration in the thumbnail as an option or by default.
Display video duration in video details.

Current behavior: can't get the video duration in the extension.
Unfortunate, as video duration is one of the main criteria for choosing what video to watch.

### Who will benefit from this feature?

Everyone using the YouTube extension.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by madza91**
## Description

Added video duration in the details view. Fixes #7086 

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/youtube/src/components/video.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ function ShowVideoDetails(props: VideoActionProps) {
   const { video, refresh } = props;
   return (
     <Action.Push
-      title="Show Details"
```
Comment (pernielsentikaer) [inline extensions/youtube/src/components/video.tsx:70]: This looks like some debug data?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Do you mind adding a changelog entry too

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by madza91** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> * Do you mind adding a changelog entry too
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Added! 👍 
Thanks for your reply!

**Comment by pernielsentikaer** [comment]
Can you approve this @tonka3000 🙂

**Review by tonka3000 — APPROVED** [review]
Sure @pernielsentikaer 😀

**Pull request merged**